README.org

```diff
@@ -103,7 +107,20 @@ I would love to extend Emacs-bootstrap to support other programming languages as
 If you think you've got your configuration of your favorite programming language perfectly,
 please don't hesitate to submit a PR.
 
+<<<<<<< HEAD
```

Comment:
See? This means a conflict is there and you should have fixed it.
`magit-ediff-resolve` in Emacs could have helped you resolve this. But i'll fix it this time.